force-app/main/default/classes/SOQL.cls

```diff
@@ -143,15 +186,20 @@ public with sharing class SOQL {
      */
     public static List<String> getContactNamesByAccount(Id accountId) {
         // Create a list to hold the Contact LastNames
-
+        List<String> contactNames = new List<String>();
         // Query the Contacts
+        List<Contact> contacts = [
+            SELECT Id, LastName
+            FROM Contact
+            WHERE AccountId = :accountId
+        ];
         // Loop through each Contact and add the LastName to the list
-        //for () {
-        //    
-        //}
+        for (Contact cont : contacts) {
+            contactNames.add(cont.LastName);
+        }
```

Comment:
This is a use case for a SOQL for loop, but ultimately it depends on your style.